diff -ru wxWidgets-2.8.12/src/mac/carbon/fontdlgosx.mm wxWidgets-2.8.12_new/src/mac/carbon/fontdlgosx.mm
--- wxWidgets-2.8.12/src/mac/carbon/fontdlgosx.mm	2011-03-22 17:31:41.000000000 +0530
+++ wxWidgets-2.8.12_new/src/mac/carbon/fontdlgosx.mm	2016-02-04 15:06:05.000000000 +0530
@@ -154,9 +154,8 @@
 
     [fontPanel setFloatingPanel:NO] ;
     [[fontPanel standardWindowButton:NSWindowCloseButton] setEnabled:NO] ;
-
-    wxMacFontPanelAccView* accessoryView = (wxMacFontPanelAccView*) [fontPanel accessoryView] ;
-    if ( accessoryView == nil)
+    wxMacFontPanelAccView* accessoryView = nil;
+    if ( [fontPanel accessoryView] == nil || [[fontPanel accessoryView] class] != [wxMacFontPanelAccView class] )
     {
         NSRect rectBox = NSMakeRect( 0 , 0 , 192 , 40 );
         accessoryView = [[wxMacFontPanelAccView alloc] initWithFrame:rectBox];
@@ -164,6 +163,10 @@
 
         [fontPanel setDefaultButtonCell:[[accessoryView okButton] cell]] ;
     }
+    else
+    {
+ 	accessoryView = (wxMacFontPanelAccView*)[fontPanel accessoryView];
+    }
 
     [accessoryView resetFlags];
 
